src/main/java/core/basesyntax/FigureProducer.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureProducer {
+    public static final int figure = 5;
```

Comment:
```suggestion
    public static final int FIGURE_COUNT = 5;
```
we declare constant in upper case
do we use this constant only in this class, if yes let's make private